RELEASE/scripts/autoscend/auto_cooking.ash

```diff
@@ -611,6 +606,52 @@ void consumeStuff()
 	}
 }
 
+boolean consumeFortune()
+{
+	if (contains_text(get_counters("Fortune Cookie", 0, 200), "Fortune Cookie"))
+	{
+		return false;
+	}
+
+	// Don't get lucky numbers for the first semi-rare if we still need to adventure in the outskirts
+	if (my_turncount() < 80 && (internalQuestStatus("questL05Goblin") < 1 && item_amount($item[Knob Goblin encryption key]) < 1))
+	{
+		return false;
+	}
+
+	if (my_level() > 12)
```

Comment:
Why no fortunes over level 12?
Also this breaks BIG path where you start at level 15.

Reply:
this was and old check we can remove it

Reply:
Sounds good, lets remove it then